pyuvdata/uvbeam/cst_beam.py

```diff
@@ -135,7 +137,11 @@ def read_cst_beam(
         fix_auto_power : bool
             For power beams, if auto polarization beams with imaginary values are found,
             fix those values so that they are real-only in data_array.
-
+        thetaphi_to_azza_fnc : callable
+            Since CST can output theta/phi coordinates in a number of conventions, you
+            can provide a function to convert from the input theta/phi to azimuth/
+            zenith-angle, as required by UVBeam. The signature of the function should
+            be ``az, za = fnc(theta, phi)``. Theta and Phi will be in radians.
```

Comment:
There should be a blank line at the end of this docstring. See examples here: https://numpydoc.readthedocs.io/en/latest/format.html